metadata/virtual_container_host.go

```diff
@@ -36,13 +37,18 @@ type VirtualContainerHostConfigSpec struct {
 	// The base config for the appliance. This includes the networks that are to be attached
 	// and disks to be mounted.
 	// Networks are keyed by interface name
-	ExecutorConfig
-
-	// The set of components to launch
-	ComponentConfig []SessionConfig
-
+	ExecutorConfig `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
This is an annotation rather than compiled code - I don't think you can use a constant here, but I'll confess I've never tried it.... and brief experimentation shows that consts are not interpreted in annotations:
https://play.golang.org/p/7l8EWR2XuJ

It's also worth noting that while all the versions are the same currently they will not increment globally - it's tied to the semantic meaning of each field.